docs/rackhd/install_os.rst

```diff
@@ -23,7 +23,7 @@ CoreOS        Graph.InstallCoreOS  899.17.0
 OS Installation Workflow APIs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Start an OS installation workflow:
+Start an OS installation workflow for CentOS:
```

Comment:
No, I just noticed that the graph name in example is InstallCentOS, so maybe the sentence should be 'An example of starting an OS installation workflow for CentOS‘